homeassistant/components/sensor/envirophat.py

```diff
@@ -157,38 +161,37 @@ def __init__(self, use_leds):
         self.voltage_1 = None
         self.voltage_2 = None
         self.voltage_3 = None
+        self.update()
```

Comment:
Yes. The True option on the end will make that we call update before we add this into state machine.